theme/locales/en.default.json

```diff
@@ -118,7 +119,7 @@
         "aria_label": "See more collections"
       }
     },
-    "product_info_bar": {
+    "add_to_cart_container": {
```

Comment:
Changed name because I made this into a new snippet, 'add_to_cart_container' because I am reusing it in several areas because its position changes on mobile vs desktop